src/main/java/com/kkm/kkm_server_v2/domain/user/domain/User.java

```diff
@@ -64,6 +69,14 @@ public void addPost(Post post) {
     @OneToMany(mappedBy = "agent", cascade = CascadeType.ALL, orphanRemoval = true)
     private List<Jjam> jjamList;
 
+    public List<Post> getJjamPostList() {
+        List<Post> postList = new ArrayList<>();
+        for (Jjam jjam : getJjamList()) {
+            postList.add(jjam.getPost());
+        }
+        return postList;
+    }
```

Comment:
```suggestion
    public List<Post> getJjamPostList() {
        return getJjamList().map(Jjam::getPost).collect(Collectors.toList());
    }
```
간결한 코드를 작성해보는 건 어떨까요

Reply:
오 좋아요